### PR TITLE
Updated Glider CUIDs

### PR DESCRIPTION
**Pull request by IDCalTech**

### CODE REVIEW

File: deployment/CP05MOAS-GL336_Deploy.csv [hunk 1]
```diff
@@ -11,45 +11,45 @@ CT20160808,,TS20160818,,CP05MOAS-GL336-02-FLORTM000,2,1,2016-08-09T13:54:00,2016
...
 TS20190516,,UNRECOVERED,,CP05MOAS-GL336-02-FLORTM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-03206,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
 TS20190516,,UNRECOVERED,,CP05MOAS-GL336-03-CTDGVM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09060,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
 TS20190516,,UNRECOVERED,,CP05MOAS-GL336-04-DOSTAM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00542,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
-TS20190516,,UNRECOVERED,,CP05MOAS-GL336-05-PARADM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50163,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
\ No newline at end of file
+TS20190516,,UNRECOVERED,,CP05MOAS-GL336-05-PARADM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50163,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL336_Deploy.csv:55]: looks good - weird highlighted space at end of the deployment 4 lines but I think it's ok.

File: deployment/CP05MOAS-GL339_Deploy.csv [hunk 2]
```diff
@@ -64,4 +64,4 @@ TS20200613,,MZ20200731,,CP05MOAS-GL339-01-ADCPAM000,11,1,2020-06-13T20:09:37,202
 TS20200613,,MZ20200731,,CP05MOAS-GL339-02-FLORTM000,11,1,2020-06-13T20:09:37,2020-07-31T18:55:48,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-FLORTM-04467,40.08153,-70.3674,,200,200,EB
 TS20200613,,MZ20200731,,CP05MOAS-GL339-03-CTDGVM000,11,1,2020-06-13T20:09:37,2020-07-31T18:55:48,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-CTDGVM-09363,40.08153,-70.3674,,200,200,EB
 TS20200613,,MZ20200731,,CP05MOAS-GL339-04-DOSTAM000,11,1,2020-06-13T20:09:37,2020-07-31T18:55:48,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-DOSTAM-00361,40.08153,-70.3674,,200,200,EB
-TS20200613,,MZ20200731,,CP05MOAS-GL339-05-PARADM000,11,1,2020-06-13T20:09:37,2020-07-31T18:55:48,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50238,40.08153,-70.3674,,200,200,EB
\ No newline at end of file
+TS20200613,,MZ20200731,,CP05MOAS-GL339-05-PARADM000,11,1,2020-06-13T20:09:37,2020-07-31T18:55:48,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00339,CGINS-PARADM-50238,40.08153,-70.3674,,200,200,EB
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL339_Deploy.csv:67]: looks good

File: deployment/CP05MOAS-GL374_Deploy.csv [hunk 3]
```diff
@@ -1,31 +1,31 @@
...
 SI20170418,,UNRECOVERED,,CP05MOAS-GL374-02-FLORTM000,5,1,2017-04-20T00:51:00,2017-04-20T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00374,CGINS-FLORTM-04468,40.02882,-70.48575,,200,200,EB;glider lost at sea
 SI20170418,,UNRECOVERED,,CP05MOAS-GL374-03-CTDGVM000,5,1,2017-04-20T00:51:00,2017-04-20T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00374,CGINS-CTDGVM-09361,40.02882,-70.48575,,200,200,EB;glider lost at sea
 SI20170418,,UNRECOVERED,,CP05MOAS-GL374-04-DOSTAM000,5,1,2017-04-20T00:51:00,2017-04-20T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00374,CGINS-DOSTAM-00012,40.02882,-70.48575,,200,200,EB;glider lost at sea
-SI20170418,,UNRECOVERED,,CP05MOAS-GL374-05-PARADM000,5,1,2017-04-20T00:51:00,2017-04-20T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00374,CGINS-PARADM-50240,40.02882,-70.48575,,200,200,EB;glider lost at sea
\ No newline at end of file
+SI20170418,,UNRECOVERED,,CP05MOAS-GL374-05-PARADM000,5,1,2017-04-20T00:51:00,2017-04-20T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00374,CGINS-PARADM-50240,40.02882,-70.48575,,200,200,EB;glider lost at sea
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL374_Deploy.csv:31]: looks good

File: deployment/CP05MOAS-GL376_Deploy.csv [hunk 4]
```diff
@@ -1,52 +1,52 @@
...
+AR24,,SI20180126,,CP05MOAS-GL376-00-ENG000000,8,1,2017-10-28T21:57:00,2018-01-27T22:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGCON-GLDRCP-00376,40.03263333,-70.6204,,1000,1000,FZ-1
+AR24,,SI20180126,,CP05MOAS-GL376-01-ADCPAM000,8,1,2017-10-28T21:57:00,2018-01-27T22:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-ADCPAM-49016,40.03263333,-70.6204,,1000,1000,FZ-1
+AR24,,SI20180126,,CP05MOAS-GL376-02-FLORTM000,8,1,2017-10-28T21:57:00,2018-01-27T22:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-FLORTM-03884,40.03263333,-70.6204,,1000,1000,FZ-1
+AR24,,SI20180126,,CP05MOAS-GL376-03-CTDGVM000,8,1,2017-10-28T21:57:00,2018-01-27T22:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-CTDGVM-09055,40.03263333,-70.6204,,1000,1000,FZ-1
+AR24,,SI20180126,,CP05MOAS-GL376-04-DOSTAM000,8,1,2017-10-28T21:57:00,2018-01-27T22:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-DOSTAM-00496,40.03263333,-70.6204,,1000,1000,FZ-1
+AR24,,SI20180126,,CP05MOAS-GL376-05-PARADM000,8,1,2017-10-28T21:57:00,2018-01-27T22:06:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00376,CGINS-PARADM-50153,40.03263333,-70.6204,,1000,1000,FZ-1
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL376_Deploy.csv:49]: looks good

File: deployment/CP05MOAS-GL379_Deploy.csv [hunk 5]
```diff
@@ -41,12 +41,12 @@ WJ20190617,,MZ20190727,,CP05MOAS-GL379-02-FLORTM000,7,1,2019-06-18T23:14:29,2019
...
+AR39,,AR41,,CP05MOAS-GL379-00-ENG000000,8,1,2019-09-27T12:22:00,2019-11-19T04:35:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGCON-GLDRCP-00379,40.0498,-70.6001,,1000,1000,FZ-1
+AR39,,AR41,,CP05MOAS-GL379-01-ADCPAM000,8,1,2019-09-27T12:22:00,2019-11-19T04:35:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-ADCPAM-50865,40.0498,-70.6001,,1000,1000,FZ-1
+AR39,,AR41,,CP05MOAS-GL379-02-FLORTM000,8,1,2019-09-27T12:22:00,2019-11-19T04:35:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-FLORTM-04201,40.0498,-70.6001,,1000,1000,FZ-1
+AR39,,AR41,,CP05MOAS-GL379-03-CTDGVM000,8,1,2019-09-27T12:22:00,2019-11-19T04:35:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-CTDGVM-09347,40.0498,-70.6001,,1000,1000,FZ-1
+AR39,,AR41,,CP05MOAS-GL379-04-DOSTAM000,8,1,2019-09-27T12:22:00,2019-11-19T04:35:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-DOSTAM-00142,40.0498,-70.6001,,1000,1000,FZ-1
+AR39,,AR41,,CP05MOAS-GL379-05-PARADM000,8,1,2019-09-27T12:22:00,2019-11-19T04:35:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00379,CGINS-PARADM-50304,40.0498,-70.6001,,1000,1000,FZ-1
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL379_Deploy.csv:49]: looks good

File: deployment/CP05MOAS-GL388_Deploy.csv [hunk 6]
```diff
@@ -1,52 +1,52 @@
...
+WJ20190617,,AR38,,CP05MOAS-GL388-00-ENG000000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGCON-GLDRCP-00388,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,AR38,,CP05MOAS-GL388-01-ADCPAM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-ADCPAM-49016,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,AR38,,CP05MOAS-GL388-02-FLORTM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-FLORTM-03190,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,AR38,,CP05MOAS-GL388-03-CTDGVM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-CTDGVM-09083,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,AR38,,CP05MOAS-GL388-04-DOSTAM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-DOSTAM-00170,39.92227,-70.91603,,1000,1000,SS-2
+WJ20190617,,AR38,,CP05MOAS-GL388-05-PARADM000,8,1,2019-06-18T22:58:15,2019-09-21T12:56:30,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00388,CGINS-PARADM-50203,39.92227,-70.91603,,1000,1000,SS-2
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL388_Deploy.csv:49]: looks good

File: deployment/CP05MOAS-GL389_Deploy.csv [hunk 7]
```diff
@@ -1,37 +1,37 @@
...
 TS20190205,,UNRECOVERED,,CP05MOAS-GL389-02-FLORTM000,6,1,2019-02-05T08:17:00,2019-05-10T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-FLORTM-04198,39.9161,-70.369722,,1000,1000,SS-2;glider lost at sea
 TS20190205,,UNRECOVERED,,CP05MOAS-GL389-03-CTDGVM000,6,1,2019-02-05T08:17:00,2019-05-10T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-CTDGVM-09349,39.9161,-70.369722,,1000,1000,SS-2;glider lost at sea
 TS20190205,,UNRECOVERED,,CP05MOAS-GL389-04-DOSTAM000,6,1,2019-02-05T08:17:00,2019-05-10T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-DOSTAM-00189,39.9161,-70.369722,,1000,1000,SS-2;glider lost at sea
-TS20190205,,UNRECOVERED,,CP05MOAS-GL389-05-PARADM000,6,1,2019-02-05T08:17:00,2019-05-10T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-PARADM-50222,39.9161,-70.369722,,1000,1000,SS-2;glider lost at sea
\ No newline at end of file
+TS20190205,,UNRECOVERED,,CP05MOAS-GL389-05-PARADM000,6,1,2019-02-05T08:17:00,2019-05-10T12:00:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00389,CGINS-PARADM-50222,39.9161,-70.369722,,1000,1000,SS-2;glider lost at sea
```
Comment (rtravis-whoi) [inline deployment/CP05MOAS-GL389_Deploy.csv:37]: looks good

File: deployment/GA05MOAS-GL364_Deploy.csv [hunk 8]
```diff
@@ -1,5 +1,5 @@
...
-NBP16-09,,UNRECOVERED,,GA05MOAS-GL364-04-CTDGVM000,2,1,2016-11-01T00:58:00,2018-01-08T05:00:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00364,CGINS-CTDGVM-09066,-42.49933333,-41.9637427,,1000,1000,Glider lost at sea
\ No newline at end of file
+NBP1609,,UNRECOVERED,,GA05MOAS-GL364-00-ENG000000,2,1,2016-11-01T00:58:00,2018-01-08T05:00:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00364,CGCON-GLDROO-00364,-42.49933333,-41.9637427,,1000,1000,Glider lost at sea
+NBP1609,,UNRECOVERED,,GA05MOAS-GL364-01-FLORDM000,2,1,2016-11-01T00:58:00,2018-01-08T05:00:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00364,CGINS-FLORDM-03123,-42.49933333,-41.9637427,,1000,1000,Glider lost at sea
+NBP1609,,UNRECOVERED,,GA05MOAS-GL364-02-DOSTAM000,2,1,2016-11-01T00:58:00,2018-01-08T05:00:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00364,CGINS-DOSTAM-00168,-42.49933333,-41.9637427,,1000,1000,Glider lost at sea
+NBP1609,,UNRECOVERED,,GA05MOAS-GL364-04-CTDGVM000,2,1,2016-11-01T00:58:00,2018-01-08T05:00:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00364,CGINS-CTDGVM-09066,-42.49933333,-41.9637427,,1000,1000,Glider lost at sea
```
Comment (rtravis-whoi) [inline deployment/GA05MOAS-GL364_Deploy.csv:5]: looks good

File: deployment/GA05MOAS-GL470_Deploy.csv [hunk 9]
```diff
@@ -1,5 +1,5 @@
...
-NBP16-09,,UNRECOVERED,,GA05MOAS-GL470-04-CTDGVM000,1,1,2016-11-01T00:36:00,2017-09-21T19:00:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00470,CGINS-CTDGVM-09150,-42.49925,-41.99616667,,1000,1000,Glider lost at sea
\ No newline at end of file
+NBP1609,,UNRECOVERED,,GA05MOAS-GL470-00-ENG000000,1,1,2016-11-01T00:36:00,2017-09-21T19:00:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00470,CGCON-GLDROO-00470,-42.49925,-41.99616667,,1000,1000,Glider lost at sea
+NBP1609,,UNRECOVERED,,GA05MOAS-GL470-01-FLORDM000,1,1,2016-11-01T00:36:00,2017-09-21T19:00:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00470,CGINS-FLORDM-03534,-42.49925,-41.99616667,,1000,1000,Glider lost at sea
+NBP1609,,UNRECOVERED,,GA05MOAS-GL470-02-DOSTAM000,1,1,2016-11-01T00:36:00,2017-09-21T19:00:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00470,CGINS-DOSTAM-00340,-42.49925,-41.99616667,,1000,1000,Glider lost at sea
+NBP1609,,UNRECOVERED,,GA05MOAS-GL470-04-CTDGVM000,1,1,2016-11-01T00:36:00,2017-09-21T19:00:00,CGVGA-05MOAS-00000,CGVEH-GLDROO-00470,CGINS-CTDGVM-09150,-42.49925,-41.99616667,,1000,1000,Glider lost at sea
```
Comment (rtravis-whoi) [inline deployment/GA05MOAS-GL470_Deploy.csv:5]: looks good

**Review by rtravis-whoi — APPROVED** [review]
Only thing of note is weird highlighted space at end of the deployment 4 lines for glider 336 deploy csv, but I think it's ok & just one of those weird quirks?

**Review by snwhite — APPROVED** [review]
All updates look good.

**Pull request merged**